apps/web/src/App.tsx

```diff
@@ -12,17 +14,25 @@ import { Router } from '@/components/router/Router'
 export const App = () => {
   const style = StyleContainer.useContainer()
 
+  function getWeb3Library(provider: any): providers.Web3Provider {
```

Comment:
we should try to have a type here (avoid adding `any` at almost all cost :), sometimes it's needed, but rarely)